site/public/js/server.js

```diff
@@ -833,8 +833,18 @@ function checkNumFilesForumUpload(input){
             
 }
 
+function replyPost(thread_id, post_id, user_id, time){
```

Comment:
Barb mentioned that she wants the reply elements to popup under the post in which we are trying to reply to. This should be changed rather than using a pop up window.